### ISSUE TITLE
emacs22 installation fails

### ISSUE THREAD

**Issue by dgutov** [rolandwalker/emacs-travis#3]
Unless you have an idea how to fix that, maybe it should be removed from the example config.

```
$ if [ "$EMACS" = "emacs22" ]; then curl -Os http://security.ubuntu.com/ubuntu/pool/universe/e/emacs22/emacs22_22.2-0ubuntu9_i386.deb && curl -Os http://security.ubuntu.com/ubuntu/pool/universe/e/emacs22/emacs22-bin-common_22.2-0ubuntu9_i386.deb && curl -Os http://security.ubuntu.com/ubuntu/pool/universe/e/emacs22/emacs22-common_22.2-0ubuntu9_all.deb && curl -Os http://security.ubuntu.com/ubuntu/pool/universe/e/emacs22/emacs22-el_22.2-0ubuntu9_all.deb && curl -Os http://security.ubuntu.com/ubuntu/pool/universe/e/emacs22/emacs22-gtk_22.2-0ubuntu9_i386.deb && sudo apt-get update -qq && sudo apt-get remove -qq emacs emacs23-bin-common emacs23-common emacs23-nox && sudo apt-get install -qq libjpeg62 xaw3dg && sudo dpkg -i emacs22_22.2-0ubuntu9_i386.deb emacs22-bin-common_22.2-0ubuntu9_i386.deb emacs22-common_22.2-0ubuntu9_all.deb emacs22-el_22.2-0ubuntu9_all.deb emacs22-gtk_22.2-0ubuntu9_i386.deb && sudo update-alternatives --set emacs22 /usr/bin/emacs22-gtk; fi

(Reading database ... 61434 files and directories currently installed.)

Removing emacs23-nox ...

emacs-remove emacs23

remove/cmake-data: Purging byte-compiled files for emacs23

emacsen-common: Handling removal of emacsen flavor emacs23

emacsen-common: purging byte-compiled files for emacs23

Removing emacs23-bin-common ...

Removing emacs23-common ...

dpkg: warning: while removing emacs23-common, directory '/usr/share/emacs23/site-lisp' not empty so not removed.

Processing triggers for man-db ...

Processing triggers for hicolor-icon-theme ...

Selecting previously unselected package libjpeg62.

(Reading database ... 58977 files and directories currently installed.)

Unpacking libjpeg62 (from .../libjpeg62_6b1-2ubuntu1_amd64.deb) ...

Selecting previously unselected package xaw3dg.

Unpacking xaw3dg (from .../xaw3dg_1.5+E-18.1ubuntu1_amd64.deb) ...

Setting up libjpeg62 (6b1-2ubuntu1) ...

Setting up xaw3dg (1.5+E-18.1ubuntu1) ...

Processing triggers for libc-bin ...

ldconfig deferred processing now taking place

Selecting previously unselected package emacs22:i386.

(Reading database ... 59022 files and directories currently installed.)

Unpacking emacs22:i386 (from emacs22_22.2-0ubuntu9_i386.deb) ...

Selecting previously unselected package emacs22-bin-common:i386.

Unpacking emacs22-bin-common:i386 (from emacs22-bin-common_22.2-0ubuntu9_i386.deb) ...

Selecting previously unselected package emacs22-common.

Unpacking emacs22-common (from emacs22-common_22.2-0ubuntu9_all.deb) ...

Selecting previously unselected package emacs22-el.

Unpacking emacs22-el (from emacs22-el_22.2-0ubuntu9_all.deb) ...

Selecting previously unselected package emacs22-gtk:i386.

Unpacking emacs22-gtk:i386 (from emacs22-gtk_22.2-0ubuntu9_i386.deb) ...

dpkg: dependency problems prevent configuration of emacs22:i386:

emacs22:i386 depends on install-info.

emacs22:i386 depends on libasound2 (>> 1.0.22).

emacs22:i386 depends on libc6 (>= 2.11).

emacs22:i386 depends on libgif4 (>= 4.1.4).

emacs22:i386 depends on libice6 (>= 1:1.0.0).

emacs22:i386 depends on libjpeg62.

emacs22:i386 depends on libncurses5 (>= 5.6+20071006-3).

emacs22:i386 depends on libpng12-0 (>= 1.2.13-4).

emacs22:i386 depends on libsm6.

emacs22:i386 depends on libtiff4.

emacs22:i386 depends on libx11-6 (>= 0).

emacs22:i386 depends on libxmu6.

emacs22:i386 depends on libxpm4.

emacs22:i386 depends on libxt6.

emacs22:i386 depends on xaw3dg (>= 1.5+E-1).

dpkg: error processing emacs22:i386 (--install):

dependency problems - leaving unconfigured

dpkg: dependency problems prevent configuration of emacs22-bin-common:i386:

emacs22-bin-common:i386 depends on emacs22-common (= 22.2-0ubuntu9).

emacs22-bin-common:i386 depends on libc6 (>= 2.4).

emacs22-bin-common:i386 depends on liblockfile1 (>= 1.0).

dpkg: error processing emacs22-bin-common:i386 (--install):

dependency problems - leaving unconfigured

Setting up emacs22-common (22.2-0ubuntu9) ...

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

install-info: warning: maintainer scripts should not call install-info anymore,

install-info: warning: this is handled now by a dpkg trigger provided by the

install-info: warning: install-info package; package emacs22-common should be updated.

Setting up emacs22-el (22.2-0ubuntu9) ...

dpkg: dependency problems prevent configuration of emacs22-gtk:i386:

emacs22-gtk:i386 depends on emacs22-bin-common (= 22.2-0ubuntu9); however:

Package emacs22-bin-common:i386 is not configured yet.

emacs22-gtk:i386 depends on install-info; however:

emacs22-gtk:i386 depends on libasound2 (>> 1.0.22); however:

emacs22-gtk:i386 depends on libc6 (>= 2.11); however:

emacs22-gtk:i386 depends on libgif4 (>= 4.1.4); however:

emacs22-gtk:i386 depends on libglib2.0-0 (>= 2.12.0); however:

emacs22-gtk:i386 depends on libgtk2.0-0 (>= 2.8.0); however:

emacs22-gtk:i386 depends on libice6 (>= 1:1.0.0); however:

emacs22-gtk:i386 depends on libjpeg62; however:

emacs22-gtk:i386 depends on libncurses5 (>= 5.6+20071006-3); however:

emacs22-gtk:i386 depends on libpng12-0 (>= 1.2.13-4); however:

emacs22-gtk:i386 depends on libsm6; however:

emacs22-gtk:i386 depends on libtiff4; however:

emacs22-gtk:i386 depends on libx11-6 (>= 0); however:

emacs22-gtk:i386 depends on libxft2 (>> 2.1.1); however:

emacs22-gtk:i386 depends on libxpm4; however:

dpkg: error processing emacs22-gtk:i386 (--install):

dependency problems - leaving unconfigured

Processing triggers for man-db ...

Errors were encountered while processing:

emacs22:i386

emacs22-bin-common:i386

emacs22-gtk:i386

The command "sudo dpkg -i emacs22_22.2-0ubuntu9_i386.deb emacs22-bin-common_22.2-0ubuntu9_i386.deb emacs22-common_22.2-0ubuntu9_all.deb emacs22-el_22.2-0ubuntu9_all.deb emacs22-gtk_22.2-0ubuntu9_i386.deb" failed and exited with 1 during install.
```
